command/server.go

```diff
@@ -50,6 +50,8 @@ import (
 
 // ServerCommand is a Command that starts the Vault server.
 type ServerCommand struct {
+	Config *server.Config
```

Comment:
This seems unnecessary, why not just give the path to storePid?

Reply:
We could. Fixed.